Clarify settings change bullets in postgres_access (#2165)
EnterpriseDB/docs: product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx

```
## Notes on the edb_admin role

- The `edb_admin` role does not have superuser priviledges by default. Contact [Support](../overview/06_support) to request superuser priviledges for `edb_admin`. If you request superuser privileges, you **must** take care to limit the number of connections used by superusers to avoid degraded service and/or compromising availability.

> - Changes to system configuration (GUCs) made by `edb_admin` or other Postgres users will not be persisted though a reboot or maintenance. Use the BigAnimal portal to modify system configuration.

Reviewer: ```suggestion
- Changes to system configuration (GUCs) made by edb_admin or other Postgres users are not persisted though a reboot or maintenance. Use the BigAnimal portal to modify system configuration.
```

Reviewer: We are updating our conventions to not use a font treatment for user names. We also avoid future case.
